server/api/audit/index.ts

```diff
@@ -0,0 +1,10 @@
+import { Express } from 'express'
+import { AuditGetFeed } from './getFeed'
+import { AuditGetLatestLogTimestamp } from './getLatestLogTimestamp'
+
+export const AuditApi = {
+  init: (express: Express): void => {
+    AuditGetFeed.init(express)
+    AuditGetLatestLogTimestamp.init(express)
```

Comment:
we better use the CRUD convention where applicable. Create/Read/Update/Delete.
the two above can be added to read?

Reply:
I agree, let's do this separately?
1 PR per section which fixes this.
Ex. Audit get it's own ticket/per which refactors to CRUD